### PR TITLE
3.6 was missing from Travis / AppVeyor build matrices.

### PR DESCRIPTION
**Pull request by johnnyreilly**
This PR adds it in in the form of TypeScript 3.6.3.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,9 +14,10 @@ install:
   - yarn lint
   - yarn add $TYPESCRIPT
 env:
+  - TYPESCRIPT=typescript@3.6.2
```
Comment (sheetalkamat) [inline .travis.yml]: suggest changing to 3.6.3 instead

Comment (johnnyreilly) [reply .travis.yml]: yup will do.  Somehow I managed to forget to add 3.6.x to the build matrix when I upgraded the comparison test pack.  Whoops.  Fixing now.  As far as I can assess there's no problems that have been introduced in the meantime, though I'm regenerating some of the project references test pack output again (which makes sense given the changes that have happened)

Comment (johnnyreilly) [reply .travis.yml]: Will push update in 15 minutes or so

Comment (johnnyreilly) [reply .travis.yml]: Done - now let's see how CI gets on

**Comment by johnnyreilly** [comment]
Hmmm.... There might be a problem with some of the project references tests:

projectReferencesRootDir:

```diff
      +[./lib/src/index.ts] A-NUMBER-OF bytes {main} [built]
      +ERROR in tsconfig.json
      +[tsl] ERROR
      +      TS5033: Could not write file '/.test/projectReferencesRootDir/lib/out/index.d.ts': Cannot read property 'set' of null.
```

File: test/comparison-tests/create-and-execute-test.js [hunk 2]
```diff
@@ -402,9 +402,9 @@ function getNormalisedFileContent(file, location) {
                 .replace(/Module build failed \(from \//gm, 'Module build failed (from ')
                 .replace(/Module Warning \(from \//gm, 'Module Warning (from ')
                 // We don't want a difference in the number of kilobytes to fail the build
-                .replace(/[\d]+([.][\d]*)? KiB/g, 'A-NUMBER-OF KiB')
+                .replace(/\s+[\d]+([.][\d]*)? KiB\s+/g, ' A-NUMBER-OF KiB ')
```
Comment (johnnyreilly) [inline test/comparison-tests/create-and-execute-test.js]: we're getting some meaningless failures here related to whitespace in webpack output.  These regex changes effectively normalize those differences.  We want meaninful failures only 😄

**Comment by sheetalkamat** [comment]
Will take a look at it shortly after setting Linux machine. I think it would be something to do with my changes since my changes override @andrewbranch ‘s work for project references ..

**Comment by johnnyreilly** [comment]
Nice - thanks!

**Comment by sheetalkamat** [comment]
@johnnyreilly I fixed this but I cant push to your branch
```
remote: Permission to TypeStrong/ts-loader.git denied to sheetalkamat.
fatal: unable to access 'https://github.com/TypeStrong/ts-loader.git/': The requested URL returned error: 403
```

**Comment by johnnyreilly** [comment]
Great work @sheetalkamat! I've just sent you an invite to join TypeStrong. Once you accept that you should be able to push (you'll be set up the same as @andrewbranch is)

**Comment by johnnyreilly** [comment]
I've seen enough passing builds to be happy - let's merge this!

**Comment by johnnyreilly** [comment]
Hey @sheetalkamat / @andrewbranch!

**Comment by johnnyreilly** [comment]
Hey @sheetalkamat / @andrewbranch!

I've merged this PR as all looked good. Interestingly, we just started to get these execution test failures with `typescript@next`:

```
2 test suite(s) failed.
 - 3.0.1_projectReferences
 - 3.6.0_projectReferencesToBeBuilt
```

Seems to be related to this:

```
TypeError: instance.compiler.getOutputPathForBuildInfo is not a function
```

Has this function been removed in the last 24 hours?

You can see this here: https://travis-ci.org/TypeStrong/ts-loader/jobs/587342045

I don't think this is a blocker for releasing - I'm planning to ship `6.1.1` shortly. But could you advise on this please?

Thanks!

**Comment by sheetalkamat** [comment]
It is... I will send a patch for this as we want to be using new apis if there.

**Comment by johnnyreilly** [comment]
Awesome - thanks!

**Pull request merged**